intranet/templates/eighth/location.html

```diff
@@ -0,0 +1,53 @@
+{% extends "page_with_nav.html" %}
+{% load phone_numbers %}
+{% load static %}
+{% load pipeline %}
+
+{% block title %}{{ block.super }} - Eighth Location{% endblock %}
+
+
+{% block css %}
+    {{ block.super }}
+    {% stylesheet 'eighth.schedule' %}
+{% endblock %}
+
+{% block js %}
+    {{ block.super }}
+{% endblock %}
+
+{% block head %}
+    {% if dark_mode_enabled %}
+        {% stylesheet 'dark/base' %}
+        {% stylesheet 'dark/nav' %}
+    {% endif %}
+{% endblock %}
+
+{% block main %}
+<div class="primary-content" id="quick-schedule">
+    <h2>Your upcoming eighth periods are:</h2>
+    <br>
+    <h3>{{ blocks.0.block_letter }} Block ({{ blocks.0.date }})</h3>
```

Comment:
To be consistent with the rest of Ion, we should hyperlink either this block information or the activity name below to the eighth signup page for this block.